utils/s3/credentials_providers/instance_profile_credentials_provider.cc

```diff
@@ -28,7 +28,7 @@ future<s3::aws_credentials> instance_profile_credentials_provider::get_aws_crede
 }
 
 bool instance_profile_credentials_provider::is_time_to_refresh() const {
-    return std::chrono::system_clock::now() >= creds.expires_at;
+    return std::chrono::steady_clock::now() >= creds.expires_at;
```

Comment:
I already commented on it in previous patch, that it can be lowres_clock. But regardless -- _here_ this change is not nice, it should be back-merged to where this method appeared for the first time